### ISSUE TITLE
powershell.scriptAnalysis.settingsPath (Description): Warning Message Should Provide A Workaround

### ISSUE THREAD

**Issue by Zian** [PowerShell/vscode-powershell#4653]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.

### Summary

The "powershell.scriptAnalysis.settingsPath" setting currently says "[This setting may not work as expected currently!](https://github.com/PowerShell/vscode-powershell/blame/0c03eb0fce3a37c3b47bad4f813dee6e47a4ce2e/package.json#L742)".

The description doesn't provide a workaround or a way to learn more about the issue. It makes me think that I will never be able to customize the PSScriptAnalyzer settings.

I would like to be able to get more information about the problem and any workarounds that might exist.

### Proposed Design

Make "This setting may not work as expected currently!" a hyperlink that goes to the corresponding issue's web page. After a quick GitHub search, it seems that https://github.com/PowerShell/vscode-powershell/issues/3168 might be the one.

If it is not a simple situation (e.g. multiple GitHub issues exist), then link to a page in the documentation that explains the matter.

**Comment by Zian** [comment]
System Information:

code --version:
1.80.0
660393deaaa6d1996740ff4880f1bad43768c814
x64

code --list-extensions --show-versions | Select-String powershell:
ms-vscode.powershell@2023.6.0

$psEditor.EditorServicesVersion says:
Major  Minor  Build  Revision
-----  -----  -----  --------
3      8      7      0


$PSVersionTable:
Name                           Value
----                           -----
PSVersion                      7.2.12
PSEdition                      Core
GitCommitId                    7.2.12
OS                             Microsoft Windows 10.0.17763
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

**Comment by andschwa** [comment]
Good idea, thanks for it. We have fixing up the PSSA settings on our short list!

**Comment by ddm-coder** [comment]
Just wanted to check in on this. A hyperlink would be helpful to shed som light on the issue and help with work arounds?
Btw, does anyone know about the root cause for this issue and if there is a potential fix planned?
